tests/ui/test_views.py

```
@@ -81,11 +81,20 @@ def test_view_parameter_notfound(client):
 
 
 def test_external_endpoint_on_institution_homepage(client, org_martigny, app):
```

Ruff (0.14.14)

[warning] Unused function argument: `org_martigny`

(ARG001)